content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,282 @@ weight: 1
 owner: istio/wg-networking-maintainers
 test: n/a
 ---
+{{< warning >}}
+Ambient is currently in alpha status, not intended for production use.
```

Comment:
We need to define a consistent way of referring to "Ambient".  We used "Ambient Mesh" in https://istio.io/latest/blog/2022/introducing-ambient-mesh/ but @howardjohn was going to try and sort out if this was right going forward.  Bringing the same up on Slack.

Reply:
I think we need to come up with a consistent way to refer to all of the ambient terms. Should they be upper or lower case, highlighted with back-ticks, etc., for terms like ztunnel, HBONE, waypoint or waypoint proxy as well as ambient itself?

I think this PR is good to merge now. We can fix up the terms and make other improvements in followup PRs.

Reply:
SGTM, thanks!